apps/forwarder-host-ovs/forwarder.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: forwarder-ovs
+  labels:
+    app: forwarder-ovs
+spec:
+  selector:
+    matchLabels:
+      app: forwarder-ovs
+  template:
+    metadata:
+      labels:
+        app: forwarder-ovs
+    spec:
+      hostPID: true
+      hostNetwork: true
+      containers:
+        - image: networkservicemeshci/cmd-forwarder-host-ovs:latest
```

Comment:
Typically we bind to particular version of a container (and they get updated from downstream).  Could that be done here?

Reply:
@pperiyasamy Look at how we are [publishing docker containers in cmd-template](https://github.com/networkservicemesh/cmd-template/blob/main/.github/workflows/docker-push-ghcr.yml).  Its appears you already have that, so all you'd need to do here is use your latest [githash](https://github.com/networkservicemesh/cmd-forwarder-ovs/pkgs/container/ci%2Fcmd-forwarder-ovs)  (currently 811ba56 I believe).  After that [update deployments](https://github.com/networkservicemesh/cmd-forwarder-ovs/blob/main/.github/workflows/update-deployments.yaml) should kick in and maintain it for you :)

Reply:
@edwarnicke ah! ok, that's awesome. Thanks. Let me fix CI for [Dockerfile.use-host-ovs](https://github.com/networkservicemesh/cmd-forwarder-ovs/blob/main/Dockerfile.use-host-ovs) in `cmd-forwarder-ovs `repo and then will update this PR.

Reply:
done. The [PR]( https://github.com/networkservicemesh/cmd-forwarder-ovs/pull/7) is also raised to address this so that update deployments would kick in automatically to change image version.